experiments/AMIP/modular/coupler_driver_modular.jl

```diff
@@ -610,20 +610,24 @@ if ClimaComms.iamroot(comms_ctx)
             T = (:regrid, :zonal_mean),
             u = (:regrid, :zonal_mean),
             q_tot = (:regrid, :zonal_mean),
-            toa = (:regrid, :horizontal_slice),
+            toa_fluxes = (:regrid, :horizontal_slice),
             precipitation = (:regrid, :horizontal_slice),
             T_sfc = (:regrid, :horizontal_slice),
+            q_liq_ice = (:regrid, :zonal_mean),
+            tubulent_energy_fluxes = (:regrid, :horizontal_slice),
         )
 
         plot_spec = (;
             T = (; clims = (190, 320), units = "K"),
             u = (; clims = (-50, 50), units = "m/s"),
-            q_tot = (; clims = (0, 50), units = "g/kg"),
-            toa = (; clims = (-250, 210), units = "W/m^2"),
-            precipitation = (clims = (0, 1e-6), units = "kg/m^2/s"),
+            q_tot = (; clims = (0, 30), units = "g/kg"),
+            toa_fluxes = (; clims = (-250, 210), units = "W/m^2"),
```

Comment:
Would symmetric `clims` spec help with interpretation here ?

Reply:
Sure, we can do that, as long as NCEP and AMIP are consistent and clear enough!